doc/api/magic-plot.md

```diff
@@ -126,37 +126,36 @@ Shortcut: `%sqlplot hist`
 
 +++
 
-Histogram does not support NULL values, so let's remove them:
+Histogram supports NULL values, by skipping them:
```

Comment:
A one-liner about with_nulls argument would be good. 
`Histogram supports NULL values, by skipping them:` this sentence can be reframed to something like `if with_nulls` argument is passed null values are ignored from the calculation`

Reply:
the histogram command by default skips the null data. It is not controlled by an argument.

Reply:
ah I see it now! It's fine then